examples/mxnet_imagenet_resnet50.py

```diff
@@ -83,32 +93,45 @@
 hvd.init()
 num_workers = hvd.size()
 
-batch_size = args.batch_size
 num_classes = 1000
 num_training_samples = 1281167
-
-data_nthreads = args.data_nthreads
-context = mx.cpu() if args.no_cuda else mx.gpu(hvd.local_rank())
-
 batch_size = args.batch_size
+epoch_size = \
+    int(math.ceil(int(num_training_samples // num_workers) / batch_size))
```

Comment:
nit: do you need to cast to `int` for the floor division? Also, maybe add `from __future__ import division` to make it python2 and python3 compatible?

Reply:
Did not think about it. Actually I did not change the formula. I think Carl also used the same [one](https://github.com/aws-samples/deep-learning-models/blob/master/models/resnet/mxnet/train_imagenet.py#L315) from Rahul.